docs/tutorials/send_receive_ingress_pem.md

```diff
@@ -0,0 +1,363 @@
+---
+title: "Exchanging messages over an ssl ingress with cert-manager's generated certs"
+description: "Steps to get a producer and a consummer exchanging messages over a deployed broker on kubernetes using an ingress and certificates genererated via cert-manager"
+draft: false
+images: []
+menu:
+  docs:
+    parent: "send-receive"
+weight: 110
+toc: true
+---
+
+### Prerequisite
+
+Before you start, you need to have access to a running Kubernetes cluster
+environment. A [Minikube](https://minikube.sigs.k8s.io/docs/start/) instance
+running on your laptop will do fine.
+
+#### Enable nginx and ssl passthrough for minikube
+
+```console
+$ minikube addons enable ingress
+$ minikube kubectl -- patch deployment -n ingress-nginx ingress-nginx-controller --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value":"--enable-ssl-passthrough"}]'
+```
+
+### Deploy the operator
+
+#### create the namespace
+
+```console
+$ kubectl create namespace send-receive-project
+$ kubectl config set-context --current --namespace=send-receive-project
+```
+
+Go to the root of the operator repo and install it:
+
+```console
+$ ./deploy/install_opr.sh
+```
+
+Wait for the Operator to start (status: `running`).
+```console
+$ kubectl get pod --namespace send-receive-project --watch
+```
+
+### Cert manager
+
+#### Install cert manager
+
+```console
+$ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.14.5/cert-manager.yaml
+```
```

Comment:
The doc could just point to cer-manager doc page to install it to avoid outdated procedures/versions in our doc